### PR TITLE
Update developer install instructions with bare metal couch install steps

### PR DESCRIPTION
**Pull request by mrjones-plip**
# Description

There was some talk in App Services about how some folks like to install outside of using docker and found that `snap` was a viable alternative.  It seemed worth documenting given the `DEVELOPMENT.md ` doc targets technical folks.

no issue

# Code review checklist
- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by mrjones-plip** [comment]
@garethbowen  - if this looks good then It'd be nice to loop in App Services as they tested CHT on snap. Otherwise, any/all feedback welcome (including this being a bad idea all together - less is sometimes more!)

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ You will need to install the following:
 - [Node.js](https://nodejs.org) 8.11.x and above LTS release (designated with an even major version number)
 - [npm](https://npmjs.com/) 6.x.x and above (to support npm ci)
 - [grunt cli](https://gruntjs.com/using-the-cli)
-- [CouchDB](https://couchdb.apache.org) 2.x ([installation instructions](http://docs.couchdb.org/en/2.3.1/install/index.html)). For simplicity we [recommend installing via docker](#couchdb-on-docker). If on a Mac, please note that installation via homebrew is **not** supported.
+- [CouchDB](https://couchdb.apache.org) 2.x ([installation instructions](http://docs.couchdb.org/en/2.3.1/install/index.html)). For simplicity we [recommend installing via docker](#couchdb-on-docker). If on a Mac, please note that installation via homebrew is **not** supported. If on Ubuntu and you don't want to use docker, see [our notes below](#couchdb-on-docker).
```
Comment (garethbowen) [inline DEVELOPMENT.md:18]: Looks like a typo in the hashtag.

```suggestion
- [CouchDB](https://couchdb.apache.org) 2.x ([installation instructions](http://docs.couchdb.org/en/2.3.1/install/index.html)). For simplicity we [recommend installing via docker](#couchdb-on-docker). If on a Mac, please note that installation via homebrew is **not** supported. If on Ubuntu and you don't want to use docker, see [our notes below](#couchdb-on-ubuntu).
```

File: DEVELOPMENT.md [hunk 2]
```diff
@@ -48,6 +48,19 @@ You can use `docker stop medic-couchdb` to stop it and `docker start medic-couch
 
 Medic recommends you familiarise yourself with other Docker commands to make docker image and container management clearer.
 
+### CouchDB on Ubuntu
+
+While we recommend use Docker to install CouchDB for development, it is still possible to install CouchDB on bare metal in Ubuntu, but there's some caveats:
```
Comment (garethbowen) [inline DEVELOPMENT.md:53]:
```suggestion
While we recommend use Docker to install CouchDB for development, it is still possible to install CouchDB on bare metal in Ubuntu, but there are some caveats: 
```

File: DEVELOPMENT.md [hunk 3]
```diff
@@ -48,6 +48,19 @@ You can use `docker stop medic-couchdb` to stop it and `docker start medic-couch
...
+
+While we recommend use Docker to install CouchDB for development, it is still possible to install CouchDB on bare metal in Ubuntu, but there's some caveats: 
+
+* For Ubuntu 18.04 and earlier, you need to specify in `apt` version to install with the `-V` flag.  For example, on a clean 18.04 install you would run:
+   ```bash
+   curl -sL https://couchdb.apache.org/repo/bintray-pubkey.asc | sudo apt-key add
+   echo "deb https://apache.bintray.com/couchdb-deb bionic main" | sudo tee -a /etc/apt/sources.list
+   apt update
+  apt install couchdb=2.3.1~bionic -V
+   ```
```
Comment (garethbowen) [inline DEVELOPMENT.md:61]: Can we fix the whitespace alignment to be consistent?

**Review by garethbowen — CHANGES_REQUESTED** [review]
Nice one. Three quick suggestions then it's good to go.

File: DEVELOPMENT.md (hunk 1 above)
Comment (mrjones-plip) [reply DEVELOPMENT.md:18]: rejecting b/c you reviewed an earlier commit (by like 38 secs ;). Was already fixed in 
b4fd150

File: DEVELOPMENT.md (hunk 3 above)
Comment (mrjones-plip) [reply DEVELOPMENT.md]: yuck!  Thanks for catching that. I'd copied the snippets from a site, tested them, and pasted them - hence the mish mash. fixed in 8884bab

**Comment by mrjones-plip** [comment]
@smbuthia - This PR is largely due to your awesome questions in Slack. Does this look like it'd be helpful for Ubuntu users like yourself looking to avoid `docker` for the time being?  Any feedback welcome before we merge to master!

**Review by garethbowen — APPROVED** [review]
Awesome, thanks!

**Comment by smbuthia** [comment]
@mrjones-plip I think the original questions were from @eliaswalyba. Nevertheless, I really like the updates.

**Comment by eliaswalyba** [comment]
Hello I was actually working some little doc improvements according to my experience with the CHT but I was not sure where exactly to PR them (in which repo).

**Pull request merged**